HonorAssistChatMessage.lua

```diff
@@ -0,0 +1,22 @@
+local addonName, addonTable = ...
+HonorAssist = addonTable
+
+-- This runs after HonorAssistDailyCalculator. That means the kill is already added to that database, so we can use the times killed from that service.
+-- When calculating realistic honor we need to decrease by 1 to get the real value of the kill. 
+ChatFrame_AddMessageEventFilter("CHAT_MSG_COMBAT_HONOR_GAIN", function(self, event, text, ...)
```

Comment:
should we add a command line switch to toggle this off and on? We can leave the global boolen in the ON state on the first use. Thoughts?

Reply:
We can but do we see people wanting to toggle a feature like this off? One thing I'm always wary of is adding toggles for a feature in an add-on since that is indicative of a loss of identity and what the product is supposed to accomplish.

Reply:
True. We just have to make sure than that the displayed honor is super accurate. :)